bee-crypto/tests/sponge.rs

```diff
@@ -1,7 +1,9 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use bee_crypto::ternary::sponge::{CurlP27, CurlP81, Kerl, Sponge, SpongeKind, UnrolledCurlP81};
+#[allow(deprecated)]
+use bee_crypto::ternary::sponge::{CurlP27, CurlP81, Kerl};
+use bee_crypto::ternary::sponge::{Sponge, SpongeKind, UnrolledCurlP81};
```

Comment:
If we deprecate all current sponge kinds, there is not reason not to deprecate `SpongeKind` ?